public/js/extraFunctions.js

```diff
@@ -1167,6 +1167,50 @@ let initialize_tableData_dropdown = function() {
   }
 }
 
+let initialize_dayOfWeek_dropdown = function() {
+  let x, i, j, selElmnt, a, b, c;
+  /* Look for any elements with the class "day_custom-select": */
+  x = document.getElementsByClassName("day_custom-select");
+  for (i = 0; i < x.length; i++) {
+    selElmnt = x[i].getElementsByTagName("select")[0];
+    /* For each element, create a new DIV that will act as the selected item: */
+    a = document.createElement("DIV");
+    a.setAttribute("class", "day_select-selected");
+    // a.innerHTML = selElmnt.options[selElmnt.selectedIndex].innerHTML;
+    x[i].appendChild(a);
+    /* For each element, create a new DIV that will contain the option list: */
+    b = document.createElement("DIV");
+    b.setAttribute("class", "day_select-items select-hide");
+    for (j = 1; j < selElmnt.length; j++) {
+      /* For each option in the original select element,
+      create a new DIV that will act as an option item: */
+      let bs;
+      if (j === 1 || j === 4) {
+        bs = "silver";
+      } else if (j === 2 || j === 5) {
+        bs = "black";
+      }
```

Comment:
What is this variable `bs` for? It doesn't seem to be used anywhere, and anyway the heavy lifting w.r.t. schedule formatting and black/silver is handled elsewhere.